### ISSUE TITLE
close(), arrow()

### ISSUE THREAD

**Issue by dbox** [jenius/axis#213]
Constantly finding myself recreating these assets over and over. So nice to have them in pure css. Should we add?

http://codepen.io/dbox/pen/bVGBow?editors=110

I have mixin super close. Just needs 1 little tweak. 

(Obviously all this will happen after #192)

**Comment by dbox** [comment]
Updated. Totally scalable now. :dancers:

**Comment by jenius** [comment]
This is amazing. 100% yes, this would save me tons of time. These looks great, and the flexibility on them is fantastic.

**Comment by dbox** [comment]
I suppose we could use it to add a fancy option to 'raquo()' too:)

**Comment by dbox** [comment]
one more.... `icon-menu()`:

http://codepen.io/dbox/pen/bVGBow

I don't really think we should add too many more, but you KNOW you're always recreating this.

**Comment by jenius** [comment]
So this one is a good idea, but as you resize the screen the lines shift around and mis-align :confused:

**Comment by dbox** [comment]
Yikes, sure enough. Odd. 

---
Daniel Box

615.414.9651 <tel:615.414.9651>
danielbox@gmail.com <http://danielbox@gmail.com/>
@dbox <http://twitter.com/dbox>

**Comment by dbox** [comment]
I think I didn't notice because have been trying out new Safari (and it doesn't do it.) 

I'll fix it up.

**Comment by dbox** [comment]
fixed :bug:

**Comment by jenius** [comment]
:+1: lgtm

**Comment by dbox** [comment]
Gunna nix the menu part. Wasn't coded the best.

Something like this is probably better anyway: https://github.com/jorenvanhee/sass-burger

I'll get arrows into docs.

**Comment by jenius** [comment]
:+1:
